pkg/metrics/eventcachemetrics/eventcachemetrics.go

```diff
@@ -39,6 +41,12 @@ var (
 		Help:        "The total of errors encountered while fetching process exec information from the cache.",
 		ConstLabels: nil,
 	}, []string{"error"})
+	ProcessCacheTotal = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace:   consts.MetricsNamespace,
+		Name:        "event_cache_process_total",
+		Help:        "The size of the process cache.",
+		ConstLabels: nil,
+	}, []string{"type"})
```

Comment:
I don't understand what's the "type" here? I would expect this metric to have no labels.

Reply:
did it because in ```FreeCache()``` pkg/process/process.go we had to reset and I think the counter would not provide the reset and If I remember I saw in some github issue for that usecase recommended to use it like this as it would provide reset option